indigo/distribution.yaml

```diff
@@ -4157,6 +4157,15 @@ repositories:
       url: https://github.com/open-rdc/icart_mini.git
       version: indigo-devel
     status: developed
+  idolink_node:
```

Comment:
You could call the repo `idolink` to match the repository name as well as the package name. The `packages` list would then be obsolete.